src/components/Common/DivIconMarker.tsx

```diff
@@ -21,14 +22,15 @@ const DivIconMarker = ({
         };
     });
 
-    const { position, eventHandlers } = marker;
+    const { position, eventHandlers, iconImg, isDay } = marker;
```

Comment:
To - do: convert this its own type in next ticket

Reply:
Gonna add this to FWM-22?

Reply:
Yep yep, I'll make the changes there